radis/lbl/base.py

```diff
@@ -1929,28 +2074,85 @@ def calc_populations_eq(self, Tgas):
         self.profiler.start("calc_eq_population", 2)
         #            printg('> Calculating equilibrium populations')
 
-        # Load partition function values
-        dgb = df1.groupby(by=["id", "iso"])
+        Qgas = {}
 
-        # ... optimize by filling all with first isotope first
-        id1, iso1 = list(dgb.indices.keys())[0]
-        molecule1 = get_molecule(id1)
-        state = self.input.state
-        parsum1 = self.get_partition_function_interpolator(molecule1, iso1, state)
-        df1["Qgas"] = parsum1.at(Tgas)
+        if "id" in df1 and "iso" in df1:
+            # Load partition function values
+            dgb = df1.groupby(by=["id", "iso"])
 
-        # ... now fill the rest:
-        for (id, iso), idx in dgb.indices.items():
-            if (id, iso) == (id1, iso1):
-                continue
+            # ... optimize by filling all with first isotope first
+            id1, iso1 = list(dgb.indices.keys())[0]
+            molecule1 = get_molecule(id1)
+            state = self.input.state
+            parsum1 = self.get_partition_function_interpolator(molecule1, iso1, state)
+            Qgas[iso1] = parsum1.at(Tgas)
```

Comment:
this "trick" was to fill the Qgas column with a first value. No, there is no more Qgas column. We can remove it